tests/unit/shared/config/ConfigTest.php

```diff
@@ -35,6 +41,8 @@ public function testGetHomeDirectoryOverriddenByCliOptions(): void {
             ->willReturn(__DIR__);
 
         $expectedDirectory = new Directory(__DIR__);
+        // make sure CLI option wins over potential environment variable
+        putenv('PHIVE_HOME=' . dirname(__DIR__));
```

Comment:
No need to actually change the environment (global state) here if properly abstracted via `Environment`.